hass.install

```diff
@@ -1,28 +1,3 @@
-HASS_USER=hass
-
-post_install() {
-  getent passwd ${HASS_USER} > /dev/null || useradd -U -s /bin/false -r ${HASS_USER} > /dev/null
-  mkdir -p /var/lib/hass
-  chown -R ${HASS_USER} /var/lib/hass
-}
-
-post_upgrade() {
-  old_version=$2
-  post_install
-  systemctl daemon-reload
-  if [[ "$old_version" < "0.24" ]]; then # TODO remove this code in while (in Sept 2016 for instance)
-    echo "Will migrate database to new format"
-    echo "stopping home-assistant if necessary"
-    systemctl is-enabled home-assistant.service > /dev/null && systemctl stop home-assistant.service
-    echo "Running: hass --script db_migrator --config /var/lib/hass/"
-    sudo -u ${HASS_USER} hass --script db_migrator --config /var/lib/hass/
-    systemctl start home-assistant.service
-  else
```

Comment:
you are deleting the mechanism to restart home assistant during package upgrade. Is this intended?
This, at least, will surprise existing users

Reply:
This is "not the arch way". You normally do not start or stop services, not even in upgrade scripts. All running services should still continue running, even after an upgrade. You'd have to manually restart them, this is something one *could* add to the upgrade note, but archlinux users should be generally aware of this. 

In general you do not want to have scripts in your package that mess up your system or change configuration.

Reply:
ok